### ISSUE TITLE
[Variants] Humans can't be overridden, causes a crash when trying to do so

### ISSUE THREAD

**Issue by Rokvach** [Regalis11/Barotrauma#4751]
Creating a variant for humans isn't supported, tested my setup by changing the inherit and speciestranslationoverride in to Moloch and it spawned a Moloch variant without any issues. If support isn't going to be added for humans maybe try and prevent the crashing somehow by telling that overriding humans isn't supported when its detected. 
```
<!--Almost everything in the character definition can be changed using the same names for the blocks.-->
<Charactervariant inherit="Human" speciesname="Humansmall" speciestranslationoverride="Human"> <!--can override the entire texture like so: texture="Content/Characters/Variants/Customcrawler.png"-->
  <health vitality="85"/>
  <!--Currently cannot change anything else than the scale-->
  <ragdoll scalemultiplier="0.05"/>
  <!--Currently can only multiply the damage. Applies to all attacks and damage types.-->
  <attack damagemultiplier="1.15"/>
</Charactervariant>
```
Console error:
![image.png](https://images.zenhubusercontent.com/5c924117534a9b0a86cdddce/f9f155c4-3553-4e5d-95c8-548bc70f302b)
Leading to crash, adding the AI element in makes it just crash at a different spot: 
![image.png](https://images.zenhubusercontent.com/5c924117534a9b0a86cdddce/d8ffcd9d-60f2-40a8-9668-485258aee6be)

**Comment by itchyOwl** [comment]
I don't think we are going to add support for humans, but we need to prevent crashing and display a warning instead.

**Comment by itchyOwl** [comment]
Fixed in Regalis11/Barotrauma-development@a2f0a8e.

**Comment by Rokvach** [comment]
Accidentally pressed "[H] Heal" on human variant and my game crashed
![image.png](https://images.zenhubusercontent.com/5c924117534a9b0a86cdddce/51e74126-e038-4c44-858d-5a8ff798aaa4)

**Comment by itchyOwl** [comment]
Both crashes fixed in Regalis11/Barotrauma-development@1ec2344.

**Comment by Rokvach** [comment]
Tested, wasn't able to get the game to crash, no issues. Closing.
